_sass/_layouts--global.scss

```diff
@@ -0,0 +1,17 @@
+
+/*
+ * The site layout is handled as a group of flex columns.
+ */
+.site {
+  display: flex;
+  min-height: 100vh;
```

Comment:
100% positive we need this? besides, min-height and vh are not good combinations sometimes we will get issues with that, make body 100% and html 100% and start from there